src/main/scala/vectorpipe/util/Caching.scala

```diff
@@ -0,0 +1,68 @@
+package vectorpipe.util
```

Comment:
@mojodna: Does 🗑️ing the 💰 imply blowing away this file?

Reply:
Yep.